src/commands/verify.ts

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
+import { CliVerifyArguments } from '../utils/define-args';
+import {
+    generateReport,
+    ValidationResult,
+    VerifyError,
+} from '../utils/generate-report';
+import { buildUrl } from '../utils/build-url';
+import { VALIDATION_ERROR_MESSAGE } from '../utils/errors';
+
+export async function verify(argv: CliVerifyArguments) {
```

Comment:
this function should throw error only if something unexpected happen. Any other "error", like failed validation, should be handled. 

The result of the validation could be communicated outside in the return value. Sth like:

```js
const verificaricationResult = await verify(...)
generateReport(outFile, verificaricationResult)

if (verificaricationResult.status === 'FAIL') {
   process.exit(1)
}
```